packages/label/src/Label.tsx

```diff
@@ -0,0 +1,50 @@
+import type React from 'react'
+
+import { LabelBase } from './LabelBase'
+
+export interface Props<C extends React.ElementType> {
+  /**
+   * The root node component. Use a string for an HTML element or a component. Defaults to "Label".
+   */
+  as?: C
+
+  /**
+   * The content of the component.
+   */
+  children?: React.ReactNode
+
+  /**
+   * The Category of the component. Defaults to "status".
+   */
+  category?: 'status' | 'product'
+
+  /**
+   * The background color of the component. Defaults to "primary".
+   */
+  variant?:
```

Comment:
I think this component could be made more flexible if we specify a `color` prop instead of `variant`. But before making that change, let's confirm with design if we want to go with the flexible/inflexible approach

Reply:
I agree, It would be better to take a color, otherwise, we would need to keep in sync any new labels that we add in SC with this.

Reply:
Updated